invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -9,7 +9,9 @@
   it under the terms of the MIT License; see LICENSE file for more details.
 #}
 
+{% if not drop_theme_basic_template %}
```

Comment:
It looks to me like a double negation `not` + `drop`. Can we convert it to positve instead?